src/assets/defaults.ts

```diff
@@ -121,7 +121,7 @@ export const defaultCustomWidgetContainers = [
 ]
 
 const hostname = window.location.hostname
-export const defaultBlueOsAddress = 'http://blueos-avahi.local'
+export const defaultBlueOsAddress = 'blueos-avahi.local'
```

Comment:
About this one, did you check if nothing breaks because of this change? I'm trying to remember but I think there a feature that expects the protocol to be on that default address (and would maybe need to be changed to respond accordingly).

Reply:
Just checked, and it was apparently used by the IFrame widget, so I've updated that one.